CHANGELOG.rst

```diff
@@ -1,6 +1,50 @@
 Changelog
 =========
 
+0.7.0
+
+MLflow 0.7.0 introduces several major features:
+
+- An R client API, available on CRAN
```

Comment:
Not on CRAN yet. In the review queue now and hopefully we'll have it by mid next week. @tomasatdatabricks